imagersite/imager_images/models.py

```diff
@@ -0,0 +1,63 @@
+"""Define image models."""
+
+from django.db import models
+from imager_profile.models import UserProfile
+
+# Create yo  models here.
+
+SHARING = (
+    ('PRIVATE', "Private"),
+    ('SHARED', "Shared"),
+    ('PUBLIC', "Public"),
+)
+
+
+class Photo(models.Model):
+    """One individaul photo."""
+
+    image_file = models.ImageField(upload_to="")
```

Comment:
Idk if you care, but did you know that you can use patterns here?  Like upload_to='images/%Y-%m-%d'